parkhangs-webapp/src/components/AddEventForm.js

```diff
@@ -51,35 +81,41 @@ class AddUpdateEventForm extends React.Component {
 
     handleAddEvent = (event) => {
         event.preventDefault()
-        const eventTimestamp = moment(`${this.state.eventTime} ${this.parsedEventDate()}`, 'hh:mm D MM YY').unix()
+        const eventTimestamp = moment(`${this.parsedEventTime()} ${this.parsedEventDate()}`, 'hh:mm D MM YY').unix()
         // const newEvent = {
         //     eventDetail: this.state.eventDetail,
         //     eventDateTime: eventTimestamp,
         // }
-        const detail =  this.state.eventDetail;
+        const detail =  this.state.eventDetail || this.eventDetail();
         const eventDateTime =  eventTimestamp;
 
         // ADD/UPDATE EVENT DISPATCH GOES HERE!
-        if (this.isNewEvent()) {
+        if (!this.props.eventId) {
            this.props.addOneEvent(this.props.parkId, detail, eventDateTime)
         }
-        // else {
-        //    this.props.updateEvent(this.props.parkId, event)
-        // }
+
+        else {
+            this.props.updateEvent(this.props.eventId, this.props.parkId, detail, eventTimestamp)
+            this.props.updatedEvent(this.props.eventId, detail, eventTimestamp)
```

Comment:
Is this duplicate too?

Reply:
no. could have named it better but it's so the parent component reflects the change.

Reply:
changed the name for clarity